src/shogun/machine/NonParametricMachine.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Yuhui Liu
+ */
+
+#ifndef NONPARAMETRCMACHINE_H_
+#define NONPARAMETRCMACHINE_H_
+
+#include <shogun/machine/Machine.h>
```

Comment:
@vigsterkr are we starting to use relative paths yet or will we do that once we switch to plugins?

Reply:
yeah at some point we should start doing that. the current include pattern we use is not the best....